svcomp-benchmark/README.md

```diff
@@ -0,0 +1,8 @@
+The [SV-benchmarks](https://github.com/sosy-lab/sv-benchmarks/tree/master/c) consist of a large number of C programs that are used as verification tasks during the [SV-COMP](http://sv-comp.sosy-lab.org/2016/benchmarks.php). We analyzed 1346 programs classified as correct from the benchmarks, and ran them after compiling with [RV-Match](http://runtimeverification.com/match). For more information on how the programs were chosen, please see the [results](results/) directory. For information on how we compiled and ran the programs with RV-Match, please see the [scripts](scripts/) directory. 
+
+We observed that 188 programs (14 %) of the programs exhibited undefined behaviors. Upon further analysis, we found that a large number of them exhibited undefined behaviors arising from subtle and seemingly innocuous mistakes. For instance, every correct program in the [floats-cdfpl](https://github.com/sosy-lab/sv-benchmarks/tree/master/c/floats-cdfpl) folder, when run after compilation with RV-Match, reported error code ["UB-CEE2"](https://github.com/kframework/c-semantics/blob/master/examples/error-codes/Error_Codes.csv) having description - "Indeterminate value used in an expression". This error occured since the main function in every program in floats-cdfpl declares a variable ```float IN```, and uses the variable in expressions without initializing it. One may choose to ignore this error, since these programs don't crash when run after compilation with gcc. A C11 compliant compiler may  however, choose to terminate the programs upon encountering undefined behavior represented by "UB-CEE2", and we'd end up with supposedly correct programs that may crash on completely valid inputs. Thus, it's imperative for programs classified to be correct to not exhibit undefined behavior. 
+
+Participants can consider using these programs to improve their tools. By identifying these undefined programs as false positives, participants may be able improve the general accuracy and effectiveness of their tools. For more information on all the undefined behaviors we discovered, please see the [results](results/) directory.
```

Comment:
The sentence "Participants can consider ..." does not sound right.  The benchmark owners should just fix it, they should either fix these programs or otherwise move them into an "undefined" category.  But it makes no sense to let undefined programs in the category of correct programs.